protocol/directory.go

```diff
@@ -15,10 +15,15 @@ type ConiksDirectory struct {
 	policies merkletree.Policies
 }
 
+type UpdatePolicies struct {
+	EpDeadline merkletree.TimeStamp
```

Comment:
I think it represents how long until the next epoch (used to set a timeout).  See the link below.